app/services/force/listing_service.rb

```diff
@@ -41,12 +41,11 @@ def self.listing(id, opts = {})
       else
         puts 'test is NOT enabled'
       end
-      
-      listing_translations = @cache.read("/ListingDetails/#{id}/translations")
-      # TODO: DAH-2451
-      # @cache.fetch("/ListingDetails/#{id}/translations") do
-      #   # call translation service
-      # end
+
```

Comment:
This change is from PR https://github.com/SFDigitalServices/sf-dahlia-web/pull/2255